app/src/main/java/com/lagradost/cloudstream3/extractors/StreamoUpload.kt

```diff
@@ -0,0 +1,47 @@
+package com.lagradost.cloudstream3.extractors
+
+import com.fasterxml.jackson.annotation.JsonProperty
+import com.lagradost.cloudstream3.app
+import com.lagradost.cloudstream3.utils.AppUtils.tryParseJson
+import com.lagradost.cloudstream3.utils.ExtractorApi
+import com.lagradost.cloudstream3.utils.ExtractorLink
+import com.lagradost.cloudstream3.utils.getAndUnpack
+import com.lagradost.cloudstream3.utils.Qualities
+import com.lagradost.cloudstream3.utils.M3u8Helper
+import org.jsoup.select.Elements
+
+class StreamoUpload1 : StreamoUpload() {
+    override val mainUrl = "https://streamoupload.xyz"
```

Comment:
legit useless class, it overrides the mainurl with the same data so in the end it just creates 2 identical classes. is this a mistake?

Reply:
done